File: lat2_c++/product.cpp
#include <iostream>
#include <string>

using namespace std;

class product{
private:
	string price;
	string idProduct;

public:
	product(){
		this->price = "";
		this->idProduct = "";
	}

	product(string price, string idProduct){
		this->price = price;
		this->idProduct = idProduct;
	}

	void setprice(string price){
		this->price = price;
	}

	void setidProduct(string idProduct){
		this->idProduct = idProduct;
	}


	string getprice(){
		return price;
	}

	string getidProduct(){
		return idProduct;
	}

	void printProduct(){
		cout << endl;
		cout << "Price : ";
		cout << getprice() << endl;
		cout << "id Prduct : ";
		cout << getidProduct() << endl; 
	}

	~product(){}
};

File: lat2_python/product.py
class product(object):

    __price = ""
    __idProduct = ""

    def __init__(self, price = "", idProduct = ""):
        self.__price = price
        self.__idProduct = idProduct

    def setprice(self, price):
        self.__price = price

    def setidProduct(self, idProduct):
        self.__idProduct = idProduct

    def getprice(self):
        return self.__price

    def getidProduct(self):
        return self.__idProduct

    def printProduct(self):
        print()
        print("Price : ", getprice())
        print("id Product : ", getidProduct())
